easybuild/easyconfigs/r/RELION/RELION-2.0.1-intel-2016b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.0.1'
+commit_id = 'a6225608e270'
```

Comment:
@hajgato hmm, shouldn't this be reflected in a `versionsuffix` (usually we go with a datestamp)?